packages/overlays/test/OverlayController.test.js

```diff
@@ -153,6 +153,30 @@ describe('OverlayController', () => {
         });
         expect(ctrl._renderTarget).to.equal(parentNode);
       });
+
+      it('throws when passing a content node that was created "offline"', async () => {
+        const contentNode = document.createElement('div');
+        const createOverlayController = () => {
+          new OverlayController({
+            ...withLocalTestConfig(),
+            contentNode,
+          });
+        };
+        expect(createOverlayController).to.throw(
+          'The contentNode you passed is not connected anywhere in the DOM',
```

Comment:
```suggestion
          'The OverlayController could not find a render target as the provided contentNode "div" is not connected to the DOM. Make sure to use "[parentNode].appendChild(contentNode)" before passing it on.',
```

Reply:
imho when you have multiple error messages in an app it's rather important to have a clear path on finding where this error belongs to

Reply:
Hmm bit confusing maybe to the average user, maybe:
```
The OverlayController could not find a render target, since the provided contentNode is not connected to the DOM. Make sure that it is connected, e.g. by doing "document.body.appendChild(contentNode)", before passing it on.
```
What do you think?

Reply:
If we do it, we should do it consistently throughout our code base, right?
In order to keep messages succint we could do:

```
[OverlayController] The contentNode you passed is (etc...)
```
?

Reply:
Probably a good idea indeed, let's start this trend here and maybe a contributor can do it everywhere else. Trend being: errors thrown should be prepended with `[ClassName] ` whenever possible (if not thrown inside a class, we can use file name or function name or something else that's descriptive)

Reply:
Changed it